tribits/examples/TribitsExampleProject/README.md

```diff
@@ -3,7 +3,7 @@
 The project `TribitsExampleProject` defines a TriBITS CMake project designed to
 provide a simple example to demonstrate how to use the TriBITS system to
 create a CMake build, test, and deployment system using a package-based
-architecture.
+architecture. To build `TribitsExampleProject` Fortran compiler is needed.
```

Comment:
To build **all** of the packages from `TribitsExampleProject`, a Fortran compiler is needed.

Reply:
Change pushed